src/Engine/ProtoScript/Runners/ProtoScriptTestRunner.cs

```diff
@@ -127,18 +127,20 @@ public bool Compile(List<ProtoCore.AST.AssociativeAST.AssociativeNode> astList,
             return buildSucceeded;
         }
 
-        public void Execute(ProtoCore.Core core, ProtoCore.Runtime.Context context)
+        public void Execute(ProtoCore.Core core, int runningBlock, ProtoCore.CompileTime.Context staticContext, ProtoCore.Runtime.Context runtimeContext)
         {
+            // Move these core setup to runtime core 
+            core.Rmem.PushFrameForGlobals(core.GlobOffset);
+            core.RunningBlock = runningBlock;
+            core.InitializeContextGlobals(staticContext.GlobalVarList);
+
+            ProtoCore.RuntimeCore runtimeCore = new ProtoCore.RuntimeCore(core.Options, core.DSExecutable, runtimeContext);
```

Comment:
This Execute function is the entrypoint of all executing DS code.
The RuntimeCore is instantiated here